### PR TITLE
Cleanup file headers

### PR DESCRIPTION
**Pull request by ndevenish**
For further discussion from #913. This cleans up the file headers, using [this script](https://gist.github.com/ndevenish/cb9ed83742d3c5102cc1aef9de1fb24a). Precisely:

- Removes all header lines except when possibly relevant (ignores e.g. `FIXME`, libtbx dispatcher instructions, pytest comment in conftest and a couple of hand-tuned files)
- Removes all shebang unless in `command_line/`, `setup.py` or `util/show_version.py`
- Makes all coding instruction `# coding: utf-8`
- Removes editor commands otherwise

### CODE REVIEW

File: util/show_version.py [hunk 1]
```diff
@@ -1,6 +1,4 @@
 #!/usr/bin/python
-#
-# this is used by the build scripts - it should not normally need to be
```
Comment (Anthchirp) [inline util/show_version.py]: bit too eager

**Comment by dagewa** [comment]
When deleting `# Author: David Waterman.` or similar, please replace with ` __author__ = "David Waterman"`. This is a guideline that comes from my employers.

**Comment by dagewa** [comment]
I'll check those guidelines are relevant for this case and report back.

**Comment by ndevenish** [comment]
I've left in the lines saying "Author: David Waterman". Does that solve your concern?

**Comment by dagewa** [comment]
I've had a reply from the IP team now, which is very reasonable. They would be satisfied if all copyright owners and authors are listed in the LICENSE file, and that this file is kept up to date. So if AUTHORS is referred to from LICENSE, all is well and I don't need any per-module claims.

I had to check - historically licensing has been a significant issue and it is better that I follow organisational guidelines stringently. This is easy to do when the guidelines are in fact quite relaxed.

**Comment by ndevenish** [comment]
Awesome - it's so much easier with sane approaches :) - and yes, much easier to keep up-to-date centrally.

Will re-clean then push.

**Comment by ndevenish** [comment]
After discussion, I've also removed the shebangs from `command_line/` - they are never (and probably shouldn't be?) called that way, only through dispatchers or `libtbx.python -m<path>`

**Pull request merged**